src/Data/Map/Internal.purs

```diff
@@ -172,9 +190,6 @@ instance traversableWithIndexMap :: TraversableWithIndex k (Map k) where
           <*> f k2 v2
           <*> traverseWithIndex f right
 
-asList :: forall k v. List (Tuple k v) -> List (Tuple k v)
-asList = identity
-
```

Comment:
Could you clarify why you're removing this?

Reply:
this using internally and in new version of `Foldable` instances this isn't necessary

Reply:
Thanks for clarifying that!